aws-xray-agent/src/main/java/com/amazonaws/xray/agent/runtime/AgentRuntimeLoader.java

```diff
@@ -21,6 +21,7 @@
 public class AgentRuntimeLoader {
     private static final String CONFIG_FILE_SYS_PROPERTY = "com.amazonaws.xray.configFile";
     private static final String CONFIG_FILE_DEFAULT_LOCATION = "/xray-agent.json";
+    private static final String CONFIG_FILE_SPRING_BOOT_LOCATION = "/BOOT-INF/classes/xray-agent.json";
```

Comment:
I never realized we support config from the classpath. For reference, OTel doesn't probably because the agent is meant to not require recompiling apps anyways.

Reply:
Oh interesting, that makes sense. We do also support filepath with the `com. amazonaws.xray.configFile` property, the classpath locations are just "good guesses" as to where it would be by default.